demo.ipynb

```diff
@@ -85,62 +113,47 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": false
+    "scrolled": false
    },
-   "outputs": [
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "Generations:   0%|          | 0/10 [00:00<?, ?it/s]\n",
-      "Gen 1 Models Fitness Eval:   0%|          | 0/20 [00:00<?, ?it/s]"
-     ]
-    },
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Genome encoding and accuracy data stored at Fri May 26 09:37:46 2017.csv \n",
-      "\n"
-     ]
-    },
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "\u001b[A"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "num_generations = 10\n",
     "population_size = 20\n",
     "num_epochs = 1\n",
     "\n",
     "devol = DEvol(genome_handler)\n",
-    "model, accurracy = devol.run(dataset, num_generations, population_size, num_epochs)\n",
-    "print model.summary()"
+    "model, accurracy, loss = devol.run(dataset, num_generations, population_size, num_epochs)\n",
```

Comment:
silly me, I didn't even look in there!   I have updated it for the return values of `devol.run` and also updated the print function to be python3 compatible. What I haven't done is the backend thing... I'll await your response to above and make those changes whatever changes you think are right.